packages/salesforcedx-vscode-lwc/package.json

```diff
@@ -23,7 +23,7 @@
   "categories": ["Languages"],
   "dependencies": {
     "@salesforce/salesforcedx-utils-vscode": "41.18.0",
-    "lwc-language-server": "0.14.2",
+    "lwc-language-server": "1.0.1",
```

Comment:
@rsalvador - This is the latest one that I can find on the registry.

Reply:
this is the one for master, for the 212 patch it should use 0.15.1